docs/deployment/local-deployment.rst

```diff
@@ -175,9 +172,89 @@ We now need to update our ``docker-compose.yml`` file, we add a new volume for t
             volumes:
                 - ./mongo/data:/data/db
 
-        blazegraph:
-            image: metaphacts/blazegraph-basic:2.2.0-20160908.003514-6
+        graphdb:
+            image: ontotext/graphdb:10.7.6
             ports:
-                - 8080:8080
+                - 7200:7200
             volumes:
-                - ./blazegraph:/blazegraph-data
+                - ./graphdb:/opt/graphdb/home
+
+GraphDB needs to have a repository set up before the FDP can interact with it. This can be done manually through the user interface, following these steps:
+
+- Start only the GraphDB container: ``docker compose up -d graphdb``
+- Navigate to your `local GraphDB instance <http://localhost:7200>`__
+- Open the ``Setup`` menu on the left, and navigate to `Repositories <http://localhost:7200/repository>`__
+- Click the `Create new repository <http://localhost:7200/repository/create>`__ button
+- Select ``GraphDB Repository``
+- Enter ``fdp`` as the ``Repository ID`` value
+- You can leave all other values to their defaults
+- Click the ``Create`` button on the bottom of the form
+
+Alternatively, these steps can be automated with the following addition to the ``graphdb`` service in our ``compose.yml`` file.
+
+.. code-block:: yaml
+
+        fdp:
+            image: fairdata/fairdatapoint:|compose_ver|
+            volumes:
+                - ./application.yml:/fdp/application.yml:ro
+            depends_on:
+                graphdb:
+                    condition: service_healthy
+
+        # ...
+
+        graphdb:
+            image: ontotext/graphdb:10.7.6
+            ports:
+                - 7200:7200
+            volumes:
+                - ./graphdb:/opt/graphdb/home
+                - ./repo.json:/tmp/repo.json:ro
+            entrypoint:
+                - bash
+                - -c
+                - |
+                  # enable bash job control
+                  set -m
+
+                  # start graphdb and move it to the background
+                  /opt/graphdb/dist/bin/graphdb &
+            
+                  # wait for 10 sec
+                  sleep 10
+            
+                  # create the repository
+                  curl -X POST http://localhost:7200/rest/repositories -H "Content-Type: application/json" -d "@repo.json"
+
+                  # move graphdb job to foreground
+                  fg
+            healthcheck:
+                test: curl http://localhost:7200/rest/repositories/fdp
```

Comment:
```suggestion
                # https://graphdb.ontotext.com/documentation/10.7/database-health-checks.html
                test: curl --fail-with-body http://localhost:7200/repositories/fdp/health || exit 1
```

Reply:
I'm getting 3 lines of log output every 5s for the `/health` endpoint. Depending on how this is deployed that'll fill a logfile fast.

Reply:
>I'm getting 3 lines of log output every 5s [...] that'll fill a logfile fast.

@kburger True, but what is the reason for choosing a 5s health check interval? (line 234)

If necessary, we could e.g. hide the response body and return only the status code (or nothing at all), something like this:

```bash
curl --no-progress-meter --output /dev/null --write-out 'status %{http_code}\n' --fail-with-body http://localhost:7200/repositories/fdp/health
```

Reply:
It's not about the curl call, but the log verbosity in graphdb. Depending on how logs are collected/stored, this adds a lot of noise. The 5s overrides the default value of 30s.